ENGAGEHF/Medications/Views/RowLabel/MedicationRowLabel.swift

```diff
@@ -0,0 +1,59 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct MedicationRowLabel: View {
+    let medication: MedicationDetails
+    @Binding var isExpanded: Bool
+    
+    
+    var body: some View {
+        HStack {
+            RecommendationSummary(medication: medication)
+            Spacer()
+            Image(systemName: "chevron.right")
+                .accessibilityLabel("Medication Expansion Button")
+                .foregroundStyle(.accent)
+                .rotationEffect(.degrees(isExpanded ? 90 : 0))
+                .animation(nil, value: isExpanded)
+        }
+        .contentShape(Rectangle())
+        .onTapGesture { isExpanded.toggle() }
```

Comment:
Also applies to some other parts of this PR:
```suggestion
            .contentShape(Rectangle())
            .onTapGesture { isExpanded.toggle() }
```